src/pages/patterns/global-header-pattern/index.mdx

```diff
@@ -0,0 +1,297 @@
+
+### Example matrix
+
+This matrix shows common system and product level processes that appear in the global and local regions.
+
+<Row>
+<Column colLg={12}>
+
+|                   | System processes                                                                                                                                   | Product processes                                                                                                                                                                                |
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Global region** | Log in, notifications, profile access, dashboard link, setting time, billing information, user permissions, switching from one product to another. | Hamburger menu to toggle local nav. Show/hide application navigation, document thumbnails. Consistent placement of local actions in common between products for example: new file, save, cancel. |
+| **Local region**  | Global settings, user rights management, notification preferences.                                                                                 | Commenting in a text doc, navigating from one peer resource to another (for example from one container, data base, or document to another), selecting modalities within your product.            |
+
+</Column>
+</Row>
+
+## Configurations
+
+The UI shell is configurable to let a product or platform choose which shell components and configurations match their user and information requirements. Simple products have the flexibility of using a header or side panel and more complex products require a combination of UI shell components to accommodate the depth of their navigation.
+
+![Example the header and left nav being used for the same IA.](images/configurations.png)
+
+<Caption>
+  A simple product navigation can be achieved with either a header or side panel
+  component.
+</Caption>
+
+### Header only
+
+The UI Shell header can be used as standalone navigation for your UI when a small number of main sections do not require a secondary navigation. The header provides a place for a site title, navigation links and dropdowns, and header icons. The header is globally persistent and will always be in view as the user navigates around the site.
+
+This configuration gives more horizontal space for the page's content and has limited room for navigation items in the header. This configuration also does not lend itself to sub-menus that need to remain open as they overlay and interfere with the page's content. Use a left panel in combination with the header if your navigation requires a sub-menu to remain open.
```

Comment:
should be
This configuration gives more horizontal space for the page’s content, but has limited room for navigation items in the header.

Reply:
add
Avoid mixing navigation links with menus in your header product navigation.